.github/workflows/release-container-image.yaml

```diff
@@ -1,14 +1,9 @@
 name: Release Container Image
 
 on:
-  schedule:
-    - cron: '3 3 * * *'
-  pull_request:
   push:
     branches: main
-    tags:
-      - 'v*.*.*'
-  workflow_dispatch:
+  workflow_call:
```

Comment:
Why `workflow_call` instead of `workflow_dispatch`?

Reply:
`worflow_call` is used so a workflow can be called by `uses` from another workflow.